sources/rest_api/config_setup.py

```diff
@@ -403,16 +450,30 @@ def create_response_hooks(
     the default behavior is to raise an HTTP error.
 
     Example:
+        def set_encoding(response, *args, **kwargs):
+            response.encoding = 'windows-1252'
+            return response
+
+        def remove_field(response: Response, *args, **kwargs) -> Response:
+            payload = response.json()
+            for record in payload:
+                record.pop("email", None)
+            modified_content: bytes = json.dumps(payload).encode("utf-8")
+            response._content = modified_content
+            return response
+
```

Comment:
I suggest to go ahead with this API and think about an additional API too, maybe close to what was implemented here:

Reply:
These hooks are really escape hatches and intended to be used for low level access to request. For sure operations on the item fields or items should be done at higher abstraction level.

Reply:
Good. I'd keep the example here so that users see the escape ladder. Once we have the better interface we should change the example and do only very low-level stuff like messing with encoding or headers.